aml-development/ozp-center
# 20180327 validate category

AMLOS-601

Description
Create custom Validate category button

Acceptance Criteria
A validate button appears on the Center Settings - Category grid.  
The validate buttons toggles enable/disable when a category is selected/deselected.


How to test code

- Pull branch 20180327_ validate_category
- Launch Application 
- Sign in as bigbrother
- Navigate to Menu ->Manage: Center Settings
- 'Validate' button appears on the toolbar of the Categories grid
- Select and deselect a category and the validate button toggles (enabled/disabled).



Please Review
@aml-development/ozp-frontend-team

**Checklist**
